CMakeLists.txt

```diff
@@ -60,10 +60,12 @@ set(FlatHash_SRCS
 )
 
 set(FlatBuffers_Tests_SRCS
+  # FLATBUFFERS_TRACK_VERIFIER_BUFFER_SIZE macro based compilation does not happen
```

Comment:
Any more detail under what circumstances this happens? Compilation units should be independent, so this is really weird.